sagemaker_model_monitor/data_quality/nlp/data_drift_detection_mmd.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        role=role,  # Iam role used in training job to access AWS ressources, e.g. S3
</pre><p>Branding: <code>Iam</code> -&gt; <code>IAM</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>